src/Transport/Topology/PreStartupChecks/TopicPartitioningCheckForForwardingTopology.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace NServiceBus.AzureServiceBus
+{
+    using System.Threading.Tasks;
+    using NServiceBus.Settings;
+    using NServiceBus.Transports;
+
+    class TopicPartitioningCheckForForwardingTopology
```

Comment:
Why is this checked required, why would you disable partitioning on a forwarding topology?